app/views/play/level/tome/SpellView.coffee

```diff
@@ -1081,7 +1081,7 @@ module.exports = class SpellView extends CocoView
     @spell.hasChangedSignificantly @getSource(), null, (hasChanged) =>
       return if hasChanged
       if e.problem.type is 'runtime'
-        @spellThang.castAether?.addProblem e.problem
+        @spellThang?.castAether?.addProblem e.problem
```

Comment:
Is this bad enough that we want to track some errors about it? With `console.error` it will show up in our logs so we can look for it in the future.

Reply:
@codeluggage oh, you are really cool!